public/about/about.pug

```diff
@@ -1,30 +1,67 @@
+    .text-serif(style="text-align: left; color: black;")
+      p Copyright to this website is held by Stanford University for all its contents with the exception of:
+      p illustrations and maps credited to third party sources,
+      p and the texts of the Explorer’s entries which are reproduced from 
+        i A Dictionary of British and Irish Travelers in Italy 1701-1800 
+        | (compiled by John Ingamells from the Brinsley Ford Archives at the Paul Mellon Center in London, YUP 1997)
+      p ISBN: XXX-X-XX-XXXXXX-X
+      p DOI: XX.XXXX/XXXXXX
+      p OCLC: XXXXXXXXXX
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Suggested Citation
+    .text-serif(style="text-align: left; color: black;")
+      p ‘The Grand Tour Explorer,’ created by Giovanna Ceserani and Giorgio Caviglia, Part 3 of Giovanna Ceserani, 
+        i A World Made by Travel
+        | , 
+        a(href="https://www.worldmadebytravel.org/thegrandtourexplorer") https://www.worldmadebytravel.org/thegrandtourexplorer
+        | , published by Center for Spatial and Textual Analysis (CESTA), Stanford 2019.
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") Creative Commons License
+    .text-serif(style="text-align: left; color: black;")
+      p The website's content, including text, user interface design, style, and imagery—unless otherwise indicated—are licensed under the creative Commons License CC BY-NC-ND 4.0. For more information and the complete text of the license please go to XXXXXX.
+      p Reproduction of material under license CC BY-NC-ND 4.0 must meet the following requirements:
+      p XXXXXX
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") By - Include the Following Attribution:
+    .text-serif(style="text-align: left; color: black;")
+      p Reprinted from The Grand Tour Explorer, created by Giovanna Ceserani and Giorgio Caviglia, Part 3 of Giovanna Ceserani, 
+        i A World Made by Travel
+        | , 
+        a(href="https://www.worldmadebytravel.org/thegrandtourexplorer") https://www.worldmadebytravel.org/thegrandtourexplorer
+        | , published by Center for Spatial and Textual Analysis (CESTA) 0 Stanford 2019. All rights reserved. Licensed under the Creative Commons License CC BY-NC-ND 4.0, 
+        a(href="https://creativecommons.org/licenses/by-nc-nd/4.0/") https://creativecommons.org/licenses/by-nc-nd/4.0/
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") NC - Non-commercial Purpose or Use:
+    .text-serif(style="text-align: left; color: black;")
+      p The content or any portion of this website cannot be sold, redistributed, linked, and/or licensed for money, used for promotional purpose, and/or included in works available for sale or licensing, even by non-profit entities.
+
+  h2.text-serif(style="font-size: 28px; text-align: center; color: #d6bc73;") ND - No-derivative:
```

Comment:
It might be easier to make this into a class (`gold-heading`) that has these css attributes (`font-size: 28px; text-align: center; color: #d6bc73;`)